testserver/settings.py

```diff
@@ -70,6 +70,8 @@
     'django.template.loaders.app_directories.Loader',
 )
 
+EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
```

Comment:
Do you know how we can actually tie into some SMTP server? While it is OK to look at the console, I'd like us to manually test actually receiving the email in various clients. Maybe we should look to see if we could use our personal gmail accounts as a SMTP server?!?